copier/vcs.py

```diff
@@ -138,7 +147,14 @@ def clone(url: str, ref: OptStr = None) -> str:
     _clone = git["clone", "--no-checkout", url, location]
     # Faster clones if possible
     if GIT_VERSION >= Version("2.27"):
-        _clone = _clone["--filter=blob:none"]
+        file_url = re.match("(file://)?(.*)", url).groups()[-1]
+        if is_git_shallow_repo(file_url):
+            warn(
```

Comment:
Please can you add a new warning based on this one and use it? That helps assertions in tests

https://github.com/copier-org/copier/blob/f8ae8d6e8ed8ec5333abe4b20ad53bc87123bff0/copier/errors.py#L102

Reply:
Sure, done